From f4b1881911976cf39df218cdae6233ca72023276 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Tue, 6 Sep 2022 07:43:33 +0800
Subject: [PATCH 50/60] Revert "drm/vc4: hdmi: Disable audio if dmas property
 is present but empty"

This reverts commit de63dbc296816aa9baf503dc1166c26c3b120503.
---
 drivers/gpu/drm/vc4/vc4_hdmi.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/vc4/vc4_hdmi.c b/drivers/gpu/drm/vc4/vc4_hdmi.c
index d5822878b5a7e..a308f2d05d173 100644
--- a/drivers/gpu/drm/vc4/vc4_hdmi.c
+++ b/drivers/gpu/drm/vc4/vc4_hdmi.c
@@ -1231,12 +1231,12 @@ static int vc4_hdmi_audio_init(struct vc4_hdmi *vc4_hdmi)
 	struct snd_soc_card *card = &vc4_hdmi->audio.card;
 	struct device *dev = &vc4_hdmi->pdev->dev;
 	const __be32 *addr;
-	int index, len;
+	int index;
 	int ret;
 
-	if (!of_find_property(dev->of_node, "dmas", &len) || !len) {
+	if (!of_find_property(dev->of_node, "dmas", NULL)) {
 		dev_warn(dev,
-			 "'dmas' DT property is missing or empty, no HDMI audio\n");
+			 "'dmas' DT property is missing, no HDMI audio\n");
 		return 0;
 	}
 
-- 
2.25.1

